docker/openemr/flex-3.17/opcache.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
+set -e
+
+if [ ! "$OPCACHE_ON" == 1 ]; then
```

Comment:
we will be assuming this is always on, so should not need this block

Reply:
Honestly the script shouldn't check for xdebug at all, it should Just Run Opcache. The test about xdebug or opcache can be handled in openemr.sh .

Reply:
i think the issue is that:
1. The docker will always be running opcache no matter what
2. The docker can only run the jit if the xdebug is turned off. And the jit requires new php.ini settings to be inserted.